js/app.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function() {
+
```

Comment:
zobacz tez jak uzywa sie jQuery w notacji literałowej, jQuery literal objects pattern http://rmurphey.com/blog/2009/10/15/using-objects-to-organize-your-code lepiej sotsowac to podejscie